src/helpers/HttpStatusCode.ts

```diff
@@ -0,0 +1,409 @@
+/**
```

Comment:
This is a lot of information we may never use. Is there not a dependency that includes type codes so we don't have to define them ourselves?

Reply:
There are several, but none as complete as I wanted. The top two results on NPM didn't include HTTP 530, which I have observed from `findRoom` myself.

This file is largely copypasta from other projects of mine. At some point, I plan to either deploy it as an NPM package, or contribute to an already-popular package.

Reply:
It'd be real nice of Node packaged constants for at least the standard HTTP codes and their names, but alas, they do not.

Reply:
Also, I like my hover text to be descriptive, rather than needing to google a ref for HTTP status codes, lol